templates/service.yaml

```diff
@@ -13,3 +15,7 @@ spec:
       port: {{ .Values.mpcPort }}
     - name: shard
       port: {{ .Values.shardPort }}
+
+{{- else }}
+# ignore this file
+{{- end }}
```

Comment:
is there a better way that we could use in the future to deal with this problem? seems like adding ifs to every template is not very scalable

Reply:
Not sure, I haven't explored Helm a whole lot. 

Maybe with sub-charts? We could have a separate project just for the report collector and another for helpers, both depending on some uber ipa chart. I'm not quite sure how to do that.